### PR TITLE
Parsing for the new create-outcome JSON response

### PR DESCRIPTION
**Pull request by stef359**
https://app.shortcut.com/clarkcan/story/31769/change-create-learning-outcome-response-to-be-json-object

### CODE REVIEW

**Review by pzalep1 — APPROVED** [review]

**Review by johngonz0 — APPROVED** [review]

**Review by Greek — APPROVED** [review]

**Review by curlyLasagna — APPROVED** [review]

**Comment by Greek** [comment]
made an accidental merge for my PR, plz ignore. i'm still gonna wait until this is merged to go ahead

**Pull request merged**
